tests/e2e/Services/Projects/ProjectsConsoleClientTest.php

```diff
@@ -19,8 +19,10 @@ class ProjectsConsoleClientTest extends Scope
     use ProjectsBase;
     use ProjectConsole;
     use SideClient;
+    use ProjectsDevelopmentKeys;
 
```

Comment:
Why do we need this here?

Reply:
As this code is modular, I'm also keeping the E2E tests as separate trait so that we can enable/disable these tests easily based on having/not having the module enabled in the BE